src/server/index.js

```diff
@@ -9,7 +9,10 @@ const wrapApplication = ( application, { PORT, logger } ) => {
 		server,
 		listen: connected => {
 			logger.info( 'Starting server on ' + PORT );
-			server = app.listen( PORT, connected );
+			server = app.listen( PORT, () => {
+				logger.info( 'Server listening on port ' + PORT );
+				connected && connected();
```

Comment:
It would be best to check that `connected` is a `function` before calling (as code could pass in anything and we'd trigger a fatal otherwise).